Assets/_Scenes/MainScene.unity

```diff
@@ -203,6 +203,80 @@ Transform:
   m_Children: []
   m_Father: {fileID: 0}
   m_RootOrder: 2
+--- !u!1 &614197765
```

Comment:
If the file `/Assets/_Scenes/MainScene.unity` is used only for camera app, it should be named `CameraScene.unity`.